qadi.py

```
@@ -125,12 +126,71 @@ async def run_single_llm_qadi(prompt: str):
     except Exception as e:
         print(f" ❌ error: {e}")
 
+def show_help():
```

Ruff (0.12.2)

Missing return type annotation for public function `show_help`

Add return type annotation: `None`

(ANN201)